site/soc/software/charts/ucp/core/postgresql.yaml

```diff
@@ -31,7 +31,7 @@ data:
   values:
     storage:
       pvc:
-        size: 2Gi
+        size: {{ volume_size }}
```

Comment:
should we have a different size var for postgres vs RabbitMQ?

Reply:
because of different namespaces ?  or each service will have different var for size ?

Reply:
sorry I should be more clear. I think db and rabbitmq are different apps, I don't why they would always have the same storage size .

Reply:
ok fixed it.